src/api/custom-resolvers/mutations/account-sign-in/account-checks/index.ts

```diff
@@ -35,15 +34,12 @@ const accountChecks = async (context: Context, account: Account, urlOrigin: stri
     console.info('Signing in account - checking account');
 
     const { id: accountId, email } = account;
+    const { isInactive } = account.status;
 
     if (!account.status.isVerified) {
       console.info('Unable to sign in account - account has not been verified yet');
 
-      const now = new Date();
-
-      const verificationHasExpired = isAfter(now, account.verificationExpiry);
-
-      if (account.verificationHash && !verificationHasExpired) {
+      if (account.verificationHash && !isInactive) {
```

Comment:
What's the reason for removing the `verificationHasExpired` check?

Should we not keep this, Otherwise if a verification has expired, a user will be able to verify their account with an invalid verification expiry time.

Reply:
Need to update the documentation